src/app/modules/item/pages/item-task-edit/item-task-edit.component.scss

```diff
@@ -0,0 +1,12 @@
+:host {
+  .iframe-container {
+    position: relative;
+    background-color: white;
+    iframe {
+      display: block;
+      width: 100%;
+      border: none;
+      min-height: 100%;
+    }
+  }
+}
```

Comment:
I don't see a reason to put code under :host, if we don't need to handle state of this component.

Reply:
fixed